src/openforms/forms/models/form_variable.py

```diff
@@ -212,7 +202,7 @@ def get_initial_value(self):
         if self.source == FormVariableSources.static:
             return STATIC_INITIAL_VALUES[self.initial_value]()
         else:
-            return self.initial_value or INITIAL_VALUES[self.data_type]
+            return self.initial_value or None
 
```

Comment:
How does this behave with an empty string is default value? I'm not sure how Formio saves that in the form builder.